### PR TITLE
Feature visual level and experience

### PR DESCRIPTION
**Pull request by felipecalvo**
## :warning: Requirements
* [x] https://github.com/mumuki/mumuki-styles/pull/58 for navbar styling.
* [x] https://github.com/mumuki/mumuki-styles/pull/59 for level icon truer to the original design

## :dart: Goal

Add level icon on navbar (for gamified organizations). Add level and level progress on profile.

## :camera_flash: Screenshots (with nicer icons)

image

_____

image

_____

See navbar screenshot on https://github.com/mumuki/mumuki-styles/pull/58, though fontawesome 4 icons are used there

### CODE REVIEW

**Comment by luchotc** [comment]
@felipecalvo Let me know when I should review this one :smile:

**Comment by felipecalvo** [comment]
Added a tooltip on navbar and animated progress on profile.

See https://youtu.be/xnxjRE9QCIA

Tr*vis is breaking because of master, not because of this branch

File: spec/dummy/db/schema.rb [hunk 1]
```diff
@@ -240,7 +240,7 @@
     t.boolean "dirty_by_submission", default: false
     t.integer "children_passed_count"
     t.integer "children_count"
-    t.boolean "once_completed"
+    t.boolean "once_completed", default: false
```
Comment (luchotc) [inline spec/dummy/db/schema.rb:243]: I think this one is stale

**Review by luchotc — APPROVED** [review]
Great Fe :heart:

File: spec/dummy/db/schema.rb (hunk 1 above)
Comment (felipecalvo) [reply spec/dummy/db/schema.rb:243]: I didn't mean to add this file lol

**Pull request merged**